tdex-sdk.md

```diff
@@ -209,59 +212,119 @@ const signedTx = await identity.signPset(unsignedTx);
+const LBTC = "5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225";
+const USDT = "c5870288a7c9eb5db398a5b5e7221feb9753134439e8ed9f569b0eea5a423330";
 
-// Alice starts a swap proposal 
+// Alice starts a swap proposal
 //
-// You need to create and provide an unsigned transaction that has 
+// You need to create and provide an unsigned transaction that has
 // enough inputs to cover amountToBeSent and the desired output
 const swapRequestMessage = swap.request({
   assetToBeSent: USDT,
   amountToBeSent: 300,
   assetToReceive: LBTC,
   amountToReceive: 0.05,
-  psetBase64: "..."
-})
+  psetBase64: "...",
+});
 
 //Bob parses the request and inspect the terms
 let json = Swap.parse({
   message: swapRequestMessage,
-  type: 'SwapRequest'
+  type: "SwapRequest",
 });
 
 // Bob provides the transaction with his signed inputs and outputs
 const swapAcceptMessage = swap.accept({
   message: swapRequestMessage,
-  psetBase64: "..."
+  psetBase64: "...",
 });
 
-
 //Alice can parse again the message and inspect the terms (optional)
 json = Swap.parse({
   message: swapAcceptMessage,
-  type: 'SwapAccept'
+  type: "SwapAccept",
 });
 
 // Alice adds his signed inputs to the transaction
 const swapCompleteMessage = swap.complete({
   message: swapAcceptMessage,
-  psetBase64: "..."
+  psetBase64: "...",
 });
 
-// Alice can sends the completed swap to Bob 
-// Now Bob finalize the transaction and broadcast it 
+// Alice can sends the completed swap to Bob
+// Now Bob finalize the transaction and broadcast it
+```
+
+### Wallet
 
+An easy-to-use transaction builder using to create swap transactions.
+
+#### Create a wallet from an Identity with `walletFromAddresses`
+
+```js
+// Create the identity
+const sender = new PrivateKey({
+  chain: "regtest",
+  type: IdentityType.PrivateKey,
+  value: {
+    signingKeyWIF: "cPNMJD4VyFnQjGbGs3kcydRzAbDCXrLAbvH6wTCqs88qg1SkZT3J",
+    blindingKeyWIF: "cRdrvnPMLV7CsEak2pGrgG4MY7S3XN1vjtcgfemCrF7KJRPeGgW6",
+  },
+});
+
+// Use the addresses to create the wallet
+const senderWallet = walletFromAddresses(sender.getAddresses(), "regtest");
 ```
+
+#### Create transaction using the wallet
+
+```js
+// createTx returns an empty transaction.
+const tx = senderWallet.createTx();
+
+// This will add output that send 50000 LBTC satoshis to the recipient address.
+// it will also adds the inputs according to the provided utxos
+const unsignedTx = senderWallet.buildTx(
+  tx,
+  // senderUtxos can be fetched using the explorer endpoint
+  senderUtxos,
+  // the recipient address
+  "el1qqt4rfky0utmv8durktmmg3athy4qfmtqz5xwvsnrty7zhujtxcthdqzncgk7vkfgpvm9varxetmp0mvxkl9tzgxq79n3mdn3v",
+  // the amount to send, in satoshis
+  50000,
+  // the asset hash to send
+  "5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225",
+  // the change address to use
+  sender.getNextChangeAddress().confidentialAddress
+);
+```
+
+#### Utils functions
```

Comment:
that's ok, but not th epurpose of this documentation. How a common developer will know hot to use these method ? We should put in the flow with others to shod common scenaris

Reply:
The utils functions are using mainly for frontend and as internal functions in others exported ones. For instance FetchTxHex is used in buildTx function. So that's difficult to integrate them in a "tdex scenario" in my opinion.

Reply:
What I mean is to put that in the flow like `Send a confidential transaction with Mnemonic (HD Wallet)` under `Identity`

Like `Get all transactions with Mnemonic  (HD Wallet)` where you show how to setup a Mnemonic instance